OpenIGTLinkIF/Widgets/Resources/UI/qSlicerIGTLConnectorPropertyWidget.ui

```diff
@@ -91,6 +100,41 @@
           </property>
          </widget>
         </item>
+        <item>
+         <widget class="QWidget" name="HeaderVersionWidget" native="true">
+          <property name="sizePolicy">
+           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
+            <horstretch>0</horstretch>
+            <verstretch>0</verstretch>
+           </sizepolicy>
+          </property>
+          <layout class="QHBoxLayout" name="horizontalLayout_5">
+           <item>
+            <widget class="QLabel" name="HeaderVersionLabel">
+             <property name="sizePolicy">
+              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
+               <horstretch>0</horstretch>
+               <verstretch>0</verstretch>
+              </sizepolicy>
+             </property>
+             <property name="text">
+              <string>Header version:</string>
```

Comment:
This is such an advanced option that I would not add it to the GUI (or maybe add it in an advanced section, hidden by default, along with the "Use streaming volume" checkbox).